specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2025-04-01/managedClusters.json

```diff
@@ -5922,6 +5925,29 @@
       },
       "description": "The list of available agent pool versions."
     },
+    "PodIPAllocationMode": {
+      "type": "string",
+      "enum": [
+        "DynamicIndividual",
+        "StaticBlock"
+      ],
+      "x-ms-enum": {
+        "name": "PodIPAllocationMode",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "DynamicIndividual",
+            "description": "Each node gets allocated with a non-contiguous list of IP addresses assignable to pods. This is better for maximizing a small to medium subnet of size /16 or smaller. The Azure CNI cluster with dynamic IP allocation defaults to this mode if the customer does not explicitly specify a podIPAllocationMode"
```

Comment:
The description differs slightly from what’s in the preview API version. Could you please create another PR to the dev branch to align them?

Reply:
Created a PR to fix the description in the preview API here: https://github.com/Azure/azure-rest-api-specs/pull/34919